internal/template-validator/tlsinfo/tlsinfo.go

```
@@ -171,4 +188,15 @@ func (ti *TLSInfo) CrateTlsConfig() *tls.Config {
 			return cert, nil
 		},
 	}
+
+	if !ti.sspTLSOptions.IsEmpty() {
+		tlsConfig.CipherSuites = ti.sspTLSOptions.CipherIDs()
+		minVersion, err := ti.sspTLSOptions.MinTLSVersionId()
+		if err != nil {
+			os.Exit(1)
```

Can you return the error instead?